src/go/kreq-gen/go.mod

```diff
@@ -0,0 +1,8 @@
+module kafka-request-generator
```

Comment:
i wonder if this should also be an rpk plugin

`rpk internal-type-fuzzer` by naming the output program `go build ./... -o .rpk-internal-type-fuzzer` like so then if you put it anywhere on the $PATH it works with `rpk`

the benefit of that is we can imagine exposing a longer fuzzing time - i.e.: run fuzzing of types for 1 hour and see if the parser crashes. 

once it's encapsulate din a program we can push it to CI and have it be run weekly or smth like that along the chaos testing. 

last, once we import gofakeit it can be the basis for our own internal benchmarking by having semantic loadgenerators in the future, just a thought. cc: @travisdowns and @twmb and @r-vasquez

Reply:
@emaxerrno good idea! This also looks very useful for generating requests to help us test new request memory use estimation, as we are making improvements in this area but it requires evaluating each request type independently so good testing per API endpoint is key.